services/alchemy_vision/v1.js

```diff
@@ -14,64 +14,171 @@
  * limitations under the License.
  **/
 
+// AlchemyAPI Image Analysis functions supported by this node
 var FEATURE_RESPONSES = {
   imageFaces: 'imageFaces',
   imageLink: "image",
-  imageKeywords: "imageKeywords"
+  imageKeywords: "imageKeywords",
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2569415486/issues/source?bid=3225115&fileBranchId=3297455#l22)